src/common/overlaycmn.cpp

```diff
@@ -140,6 +140,20 @@ void wxDCOverlay::Clear()
 
 #include "wx/window.h"
 
+#if defined(__WXGTK__) && !defined(__WXGTK3__)
```

Comment:
I would just put this in `Init()` itself, i.e. avoid defining the extra function which is used for wxGTK2 only but just inlined its body there inside `#if GTK2` test.

The rationale is that it's not obvious when seeing the call to `wxDontClipSubWindows()` that it only does something for GTK2, and it would be helpful to see it immediately.

This would also be a bit shorter.